infra/terraform/gcp/feast.tf

```diff
@@ -113,7 +113,7 @@ resource "helm_release" "feast" {
   depends_on = [kubernetes_secret.feast-postgres-secret, kubernetes_secret.feast_sa_secret]
 
   name  = var.name_prefix
-  chart = "../../charts/feast"
+  chart = "feast-charts/feast"
```

Comment:
Would it be easier to use a remote repository? 

Something like `chart = "https://feast-helm-charts.storage.googleapis.com/feast-0.100.4.tgz"`?

Reply:
It's easier I agree, with this doc changes are not necessary. However this solution fixes the version and with every new helm chart release you'll have to keep these terraform files up to date.

Tell me what you prefer, I'm good with both

Reply:
> However this solution fixes the version and with every new helm chart release you'll have to keep these terraform files up to date

I actually think this is a feature, not a bug :)

We'd essentially be pinning the dependencies to a specific version, which I think is ideal. We want to provide an opinionated set of components that we know work well together. So I think pointing to a specific chart makes the most sense.

We could use this argument layout to specific the repo + chart + version
```
resource "helm_release" "example" {
  name       = "my-redis-release"
  repository = "https://charts.bitnami.com/bitnami"
  chart      = "redis"
  version    = "6.0.1"
}
```

Reply:
@woop Done!